docs/user/mobile-applications.rst

```diff
@@ -195,6 +195,44 @@ In this screen, users can choose which notification types they want to receive.
   .. image:: native-applications/iOS/settings_push_detail.png
             :width: 300
 
+
+Modeling Exercises
+^^^^^^^^^^^^^^^^^^
+Users can participate in modeling exercises, view their submissions and also view the feedback they received in the Artemis iOS app.
+
+- Participate in modeling exercise:
+    #. Add UML elements to the canvas
+    #. Edit attributes and methods of elements
+    #. Draw UML relationships between elements
+    #. Edit relationship properties
+    #. Submit the diagram
+
+.. raw:: html
+
+    <iframe src="https://live.rbg.tum.de/w/artemisintro/42870?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="350">
+        Video tutorial for participating in a modeling exercise on TUM-Live.
+    </iframe>
```

Comment:
```suggestion
    .. raw:: html

        <iframe src="https://live.rbg.tum.de/w/artemisintro/42870?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="350">
            Video tutorial for participating in a modeling exercise on TUM-Live.
        </iframe>
```
This is part of the item starting with `- Participate […]`. To semantically structure the document, it should be indented accordingly.